src/model_instance.rs

```diff
@@ -99,8 +100,9 @@ impl ModelInstance {
             ffm_bit_precision: 18,
             ffm_separate_vectors: false, // DEPRECATED, UNUSED
             fastmath: true,
+			initialization_type: String::from("default"),
```

Comment:
please fix whitespace/indentation issues in this PR

Reply:
Default rustfmt introduces many changes, making the actual contribution(s) hard to read. Will open a separate one just for formatting if that's fine